src/i18n/i18n-concat.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```

Comment:
So do we want this file to eventually be a part of something like `frontend-i18n` so that we don't have to copy it around from repo to repo?

Reply:
Yes. Note that this file actually lived in reactifex repo, but we haven’t been able to take it back, so we might just need to crest a new one.